tiledb/bioimg/converters/base.py

```diff
@@ -227,6 +229,22 @@ def to_tiledb(
                 tiledb.Array.create(uri, schema)
                 uris.append(uri)
 
+                # Calculate downsample factor
+                dims_map = dict(zip([*level_axes.dims], level_shape))
```

Comment:
`zip([*level_axes.dims], level_shape)) == zip(level_axes.dims, level_shape))` (strings are iterable, no need to make it a list)